data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,689 @@ mission "Remnant: Learn Sign Follow Up"
-
-
-
-mission "Remnant: Follow-up on Terminus Researchers AB-3"
-	name "Check up on Terminus Researchers"
-	description "Returning with information on the wormhole researchers."
-	source
-		government "Deep"
-	destination
-		government "Remnant"
-	to offer
-		or
-			has "Remnant: Follow-up on Terminus Researchers A-2: done"
-			has "Remnant: Follow-up on Terminus Researchers B: done"
-	on offer
-		conversation
-			`Back on board <ship>, you take a moment to flip through the papers. You quickly realize that little beyond the abstract is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`
-			`	All the academic stuff aside, the summary on the most recent paper confirms that the researchers are making progress: they have determined that the spatial anomaly in the Terminus system is in fact a wormhole with some kind of instability. They also speculate that it may be possible for a ship to artificially stabilize the wormhole enough for it to pass through unharmed.`
-			`	Prefect Torza is definitely going to be interested in seeing this.`
+			
+			`	You approach a spaceport information desk and ask to see Prefect Torza, who you spoke to the last time you informed the Remnant about the Deep researching the wormhole. Luckily, Torza is on-world at the moment, and you are told to meet him in the cafeteria. When you arrive, you find that Torza has just finished a meal and was leaving.`
+			`	"Ah, Captain. I was told you were looking for me. We should find a private place to talk."`
+			`	You follow Torza to a similar armory as before. "Is there an update to the research on the Threshold that the Deep are doing?" he asks after sitting down.`
+			choice
+				`	"I've helped the scientists learn how to pass through the wormhole."`
+				`	"The scientists have discovered how to pass through the wormhole."`
+					goto discovered
+			
+			apply
+				set "remnant: admitted to helping deep"
+			`	Torza's eyes go wide. "Why would you do this?" he asks. "Do you not understand that our secrecy is of upmost importance?"`
+			choice
+				`	"I don't believe that they will be a threat to you."`
+					goto threat
+				`	"I'm sorry, but I helped them before coming to understand this."`
+			
+			apply
+				set "remnant: lied to torza: understanding"
+			`	"Liar!" Torza yells at you while standing up from the bench. "You told us of these scientists after we had already trusted you with a license." Torza walks away from you angrily, then turns his head.`
```

Comment:
This conversation isn't occurring through sign language. Verbiage further up in this conversations is "says," not "signs."

Reply:
Ah, I see. I still personally think using `yell` is iffy given the Remnant sing to speak instead of using normal inflexion, but it doesn't matter all that much if you all would prefer not to change it.

Reply:
It might not be ideal, but I can't think of a good alternative, and it does fit the situation. As such, I'm fine with it.

Reply:
Well I don’t see why “yelling” from the Remnant can’t be like a hefty opera-like singing.

Reply:
Sure, that's how I picture it.